.github/workflows/build-and-test-pr.yaml

```
@@ -4,28 +4,31 @@ on:
   pull_request:
 
 jobs:
-  get-modified-packages:
-    uses: autowarefoundation/autoware-github-actions/.github/workflows/reusable-get-modified-packages.yaml@tier4/proposal
-    with:
-      script-ref: tier4/proposal
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:galactic
 
-  get-modified-source-files:
-    uses: autowarefoundation/autoware-github-actions/.github/workflows/reusable-get-modified-source-files.yaml@tier4/proposal
+    steps:
+      - name: Get modified packages
+        id: get-modified-packages
+        uses: autowarefoundation/autoware-github-actions/get-modified-packages@feature/composite-action
```

TODO: Fix after the evaluation is done.

```suggestion
        uses: autowarefoundation/autoware-github-actions/get-modified-packages@tier4/proposal
```